function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -688,8 +711,8 @@ private string GetToken()
 
             string TenantId = Environment.GetEnvironmentVariable("TenantId") ?? "";
 
-            string resourceUri = Environment.GetEnvironmentVariable("ResourceUri") ?? "";
-            string AuthEndPoint = Environment.GetEnvironmentVariable("AuthEndPoint") ?? "";
+            string resourceUri = $"https://{Environment.GetEnvironmentVariable("AuthenticationUri") ?? "purview.azure.net"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com